src/firebase.js

```diff
@@ -1,7 +1,6 @@
-import firebase from "firebase";
-import 'firebase/firestore';
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/firestore';
 
-require("dotenv").config();
```

Comment:
With webpack v5, we are no longer able to access dotenv (and in turn fs) on the client side.  I am hoping we don't need to do this, and can instead rely on react-scripts to provide us with vars (the ones that start with REACT_APP_).